backend/Squirrel.Core/Squirrel.Core.BLL/Services/UserService.cs

```diff
@@ -43,6 +43,18 @@ public async Task<UserDto> GetUserByEmailAsync(string email)
         }
         return _mapper.Map<UserDto>(userEntity);
     }
+    
+    public async Task<List<UserDto>> GetAllUsersAsync()
+    {
+        var userEntities = await _context.Users.ToListAsync();
+
+        if (userEntities == null)
+        {
+            throw new EntityNotFoundException(nameof(User));
+        }
+        
+        return _mapper.Map<List<UserDto>>(userEntities);
```

Comment:
I think this method should not throw an exception: if there are no users, just return an empty list, because it is not finding, but getting users